actions/castVote.ts

```diff
@@ -41,10 +41,15 @@ export async function castVote(
   const payer = walletPubkey
   // Explicitly request the version before making RPC calls to work around race conditions in resolving
   // the version for RealmInfo
-  const programVersion = await getGovernanceProgramVersion(
+
```

Comment:
remove

Reply:
removed- will need to restore when i want to test on localnet.